### ISSUE TITLE
Build failed with source code with ppl.cv master

### ISSUE THREAD

**Issue by del-zhenwu** [open-mmlab/mmdeploy#117]
Thanks for your bug report. We appreciate it a lot.

**Checklist**

1. I have searched related issues but cannot get the expected help.
2. I have read the [FAQ documentation](https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/faq.md) but cannot get the expected help.
3. The bug has not been fixed in the latest version.

**Describe the bug**

**Reproduction**

1. What command or script did you run?

```none
A placeholder for the command.
```

2. Did you make any modifications on the code or config? Did you understand what you have modified?

**Environment**

1. Please run `python tools/check_env.py` to collect necessary environment information and paste it here.
2. You may add addition that may be helpful for locating the problem, such as
    - How you installed PyTorch [e.g., pip, conda, source]
    - Other environment variables that may be related (such as `$PATH`, `$LD_LIBRARY_PATH`, `$PYTHONPATH`, etc.)

**Error traceback**

If applicable, paste the error trackback here.

```none
A placeholder for trackback.
```

**Bug fix**

If you have already identified the reason, you can provide the information here. If you are willing to create a PR to fix it, please also leave a comment here and that would be much appreciated!

**Comment by del-zhenwu** [comment]
Issue resolved with using `ppl.cv:v1.6.1`

**Comment by RunningLeon** [comment]
> Issue resolved with using `ppl.cv: v0.6.1`

Thanks for your feedback. Will update it in document later.

**Comment by tehkillerbee** [comment]
@RunningLeon Looks like ppl:cv renamed some type names:

```
[ 95%] Building CUDA object csrc/preprocess/cuda/CMakeFiles/mmdeploy_cuda_transform_impl.dir/cast.cu.o
/home/nvidia/git/project/MMDeploy/csrc/preprocess/cuda/resize_impl.cpp: In member function ‘ppl::common::RetCode mmdeploy::cuda::ResizeImpl::DispatchImpl(Args&& ...)’:
/home/nvidia/git/project/MMDeploy/csrc/preprocess/cuda/resize_impl.cpp:48:51: error: ‘INTERPOLATION_TYPE_LINEAR’ is not a member of ‘ppl::cv’
                                          ppl::cv::INTERPOLATION_TYPE_LINEAR);
                                                   ^~~~~~~~~~~~~~~~~~~~~~~~~
/home/nvidia/git/project/MMDeploy/csrc/preprocess/cuda/resize_impl.cpp:48:51: note: suggested alternative: ‘INTERPOLATION_LINEAR’
                                          ppl::cv::INTERPOLATION_TYPE_LINEAR);
                                                   ^~~~~~~~~~~~~~~~~~~~~~~~~
                                                   INTERPOLATION_LINEAR
/home/nvidia/git/project/MMDeploy/csrc/preprocess/cuda/resize_impl.cpp:52:51: error: ‘INTERPOLATION_TYPE_NEAREST_POINT’ is not a member of ‘ppl::cv’
                                          ppl::cv::INTERPOLATION_TYPE_NEAREST_POINT);
                                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/nvidia/git/project/MMDeploy/csrc/preprocess/cuda/resize_impl.cpp:52:51: note: suggested alternative: ‘INTERPOLATION_NEAREST_POINT’
                                          ppl::cv::INTERPOLATION_TYPE_NEAREST_POINT);
                                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                   INTERPOLATION_NEAREST_POINT
[ 96%] Building CUDA object csrc/preprocess/cuda/CMakeFiles/mmdeploy_cuda_transform_impl.dir/
```

**Comment by RunningLeon** [comment]
> 
@tehkillerbee Hi, agreed. Will discuss with [OpenPPL](https://github.com/openppl-public/ppl.cv) team and lock a version if possible.

**Comment by TheSeriousProgrammer** [comment]
Hi , i guess the same issue is reflected in docker build too , can you shift the ppl.cv version used in the docker image too?

**Comment by RunningLeon** [comment]
> Hi , i guess the same issue is reflected in docker build too , can you shift the ppl.cv version used in the docker image too?(For Now) in the docker/GPU I had to do the following changes
> 
> ```
> RUN wget https://github.com/openppl-public/ppl.cv/archive/refs/tags/v0.6.1.zip && unzip v0.6.1.zip
> RUN mv ppl.cv-0.6.1 ppl.cv && \
>     cd ppl.cv &&\
>     ./build.sh cuda
> 
> #RUN git clone https://github.com/openppl-public/ppl.cv.git &&\
> #    cd ppl.cv &&\
> #    ./build.sh cuda
> ```

@TheSeriousProgrammer, Hi, thanks for your feedback. Once the version is locked, will update the dockerfile later.

**Comment by lvhan028** [comment]
> Hi , i guess the same issue is reflected in docker build too , can you shift the ppl.cv version used in the docker image too?(For Now) in the docker/GPU I had to do the following changes
> 
> ```
> RUN wget https://github.com/openppl-public/ppl.cv/archive/refs/tags/v0.6.1.zip && unzip v0.6.1.zip
> RUN mv ppl.cv-0.6.1 ppl.cv && \
>     cd ppl.cv &&\
>     ./build.sh cuda
> 
> #RUN git clone https://github.com/openppl-public/ppl.cv.git &&\
> #    cd ppl.cv &&\
> #    ./build.sh cuda
> ```

#129 reported this issue, too.
Would you like to make a pr to fix it?

**Comment by TheSeriousProgrammer** [comment]
Yes i have sent a pr at https://github.com/open-mmlab/mmdeploy/pull/142 , regarding these changes , can you check it?

**Comment by RunningLeon** [comment]
fixed in #143
